cns/middlewares/k8sSwiftV2_windows.go

```diff
@@ -5,8 +5,12 @@ import (
 	"github.com/Azure/azure-container-networking/cns/logger"
 )
 
-// setRoutes sets the routes for podIPInfo used in SWIFT V2 scenario. This is a no-op as route setting is not applicable for Windows.
-func (k *K8sSWIFTv2Middleware) setRoutes(_ *cns.PodIpInfo) error {
-	logger.Printf("[SWIFTv2Middleware] setRoutes is a no-op on Windows")
+// for AKS L1VH, do not set default route on infraNIC to avoid customer pod reaching all infra vnet services
+// default route is set for secondary interface NIC(i.e,delegatedNIC)
+func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
+	logger.Printf("[SWIFTv2Middleware] setRoutes: only set skipDefaultRoutes flag to true for InfraNIC")
```

Comment:
this is not a useful log, as it is simply describing the code (the previous log was also not a useful log).
the log should tell us _what happened_. so at a minimum it needs to be moved inside and/or after the conditional, to give us that information about what decision was made and why.